### PR TITLE
[SQL Server] (No.9) add sqlserver show-property action codes.

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    public ISqlServer execute() {
+        final Operation operation = TelemetryManager.createOperation(ActionConstants.MySQL.CREATE);
+        try {
+            operation.start();
+            final String subscriptionId = config.getSubscription().subscriptionId();
+            EventUtil.logEvent(EventType.info, operation, Collections.singletonMap(TelemetryConstants.SUBSCRIPTIONID, subscriptionId));
+            // create resource group if necessary.
+            if (config.getResourceGroup() instanceof Draft) {
+                Azure azure = AuthMethodManager.getInstance().getAzureClient(subscriptionId);
+                ResourceGroup newResourceGroup = azure.resourceGroups().define(config.getResourceGroup().name()).withRegion(config.getRegion().getName()).create();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java:53]: create group is merged in develop

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                config.setResourceGroup(newResourceGroup);
+            }
+            // create sql server
+            SqlServerEntity entity = SqlServerEntity.builder().name(config.getServerName()).subscriptionId(config.getSubscription().subscriptionId())
+                    .resourceGroup(config.getResourceGroup().name()).region(Region.fromName(config.getRegion().getName())).administratorLoginName(config.getAdminUsername())
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java:58]: wrap a help method: fromConfig

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        type = AzureOperation.Type.SERVICE
+    )
+    public ISqlServer execute() {
+        final Operation operation = TelemetryManager.createOperation(ActionConstants.MySQL.CREATE);
+        try {
+            operation.start();
+            final String subscriptionId = config.getSubscription().subscriptionId();
+            EventUtil.logEvent(EventType.info, operation, Collections.singletonMap(TelemetryConstants.SUBSCRIPTIONID, subscriptionId));
+            // create resource group if necessary.
+            if (config.getResourceGroup() instanceof Draft) {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java:51]: this piece of code shall soon be duplicate: 
config.getResourceGroup() instanceof Draft 
...

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java (hunk 1 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java:53]: good.

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java]: updated.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java (hunk 2 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java]: updated.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java (hunk 3 above)
Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/task/CreateSqlServerTask.java:51]: remember to merge them

**Review by andxu — APPROVED** [review]

**Pull request merged**